src/StatusQ/Components/StatusExpandableSettingsItem.qml

```diff
@@ -0,0 +1,155 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+Rectangle {
+    id: statusExpandableSettingsItem
+
+    property alias primaryText: primaryText.text
+    property alias secondaryText: secondaryText.text
+    property alias button: button
+    property alias expandableComponent: expandableRegion.sourceComponent
+
+    property bool expandable: true
+    property  bool expanded: false
+    property StatusIconSettings icon: StatusIconSettings {
+        color: Theme.palette.directColor1
+        background: StatusIconBackgroundSettings {
+            width: 32
+            height: 32
+            color: Theme.palette.primaryColor2
+        }
+    }
+
+    implicitWidth: 718
+
+    radius: 8
+    color: "transparent"//Theme.palette.indirectColor1
```

Comment:
The comment we should remove.

I think going with "transparent" is fine for now. If we ever run into cases where this is problematic, we can create a new `Theme.palette.statusExpandableSettingsItem` property